Documentation/ApiOverview/SiteHandling/BaseVariants.rst

```diff
@@ -24,6 +24,9 @@ Conditions are based on Symfony expression language and allow flexible condition
 
 would define a base variant to use in Development context.
 
+The YAML parser also processes some special syntax, e.g. environment variables. See :ref:`yaml-api` for further information on YAML parsing.
+That way, variants can be built from environment variables.
+
```

Comment:
```suggestion
Notice that you can refer to environment variables in the :yaml:`base` using `%env(...)%` but for `condition` you must use `getenv(...)` instead.

```

Reply:
Should that be rendered as an actual note?

Reply:
You mean an admonition? Yes, would be useful.

Reply:
Done.